### PR TITLE
Web: Add MarkInverse and ResourceLabelTooltip contents

### PR DESCRIPTION
**Pull request by kimlisa**
part of https://github.com/gravitational/teleport/issues/46976

This PR
- adds an inverse version of `Mark` to be used inside `Tooltips`, otherwise `Mark` is invisible. Tooltips uses inverse background color of main theme (eg: tooltips for dark theme uses light background and vice versa). 
- adds a single place to get tooltip content to explain user provided labels

Screenshots
RDS - dark theme
image

EKS - dark theme
image

SERVER - light theme
image

DATABASE - light theme
image

KUBE - bblp theme
image

### CODE REVIEW

**Comment by kimlisa** [comment]
if it helps to see how it will be used, here is demo

image

**Review by zmb3 — APPROVED** [review]
Looks great!

File: web/packages/design/src/Mark/Mark.tsx [hunk 1]
```diff
@@ -26,3 +26,8 @@ export const Mark = styled.mark`
   background-color: ${p => p.theme.colors.interactive.tonal.neutral[2]};
   color: inherit;
 `;
+
+export const MarkInverse = styled(Mark)`
```
Comment (flyinghermit) [inline web/packages/design/src/Mark/Mark.tsx:30]: nit: JSdoc on exported function/var/types/styles etc.

File: web/packages/teleport/src/Discover/Shared/ResourceLabelTooltip/ResourceLabelTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Link from 'design/Link';
+import { MarkInverse } from 'design/Mark';
+import { Position } from 'design/Popover/Popover';
+import { IconTooltip } from 'design/Tooltip';
+import styled from 'styled-components';
+
+export function ResourceLabelTooltip({
```
Comment (flyinghermit) [inline web/packages/teleport/src/Discover/Shared/ResourceLabelTooltip/ResourceLabelTooltip.tsx:25]: nit: JSDoc

**Review by flyinghermit — APPROVED** [review]

**Comment by thedevelopnik** [comment]
Copy looks good to me! Thanks!

**Pull request merged**